publish/tests/factories.py

```diff
@@ -31,6 +31,8 @@ class Meta:
 
     dandiset = factory.SubFactory(DandisetFactory)
     metadata = factory.Faker('pydict', value_types=['str', 'float', 'int'])
+    name = factory.Faker('pystr')
```

Comment:
I think we can get even better fake data for these if you look in https://faker.readthedocs.io/en/master/providers/faker.providers.lorem.html

Reply:
Addressed in https://github.com/dandi/dandi-publish/pull/63/commits/0b00fcd6288836f50c570e13221a01123f57d57d